tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -445,6 +445,7 @@ module.exports = {
       await this
         .useXpath()
         .getAttribute(linkSelector, 'filename', function (result) {
+          console.log(result)
```

Comment:
debug line!

Reply:
It was to reproduce the error and find where and why it actually fails.It is fixed now